vue-handler.js

```diff
@@ -171,14 +171,14 @@ function vueFilePage(data) {
 
                 <div>
                     <a class="button share-icon">
-                        <i class="fad fa-share-alt"></i> share කිරීම සඳහා link එකක් ලබාගන්න
+                        <i class="fad fa-share-alt"></i> බෙදාගැනීමට සබැඳියක් ගන්න
                     </a>
                 </div>
 
                 <div v-if="details.length" class="entry-details">{{ details }}</div>
-                <div v-if="details.length < 10" class="file-details">මේ පොත ඔබ කියවා තිබේනම් හෝ පොත ගැන යම් විස්තරයක් දන්නේ නම් 
-                    මේ පිටුවට පැමිණෙන අයට දැනගැනීම පිණිස මෙතැන පලකිරීමට සුදුසු යම් විස්තරයක් හැඳින්වීමක් ඔබට ලියා අප වෙත email කරන්න. 
-                    පොතේ link එකද සඳහන් කිරීමට අමතක කරන්න එපා. සිංහලෙන් හෝ ඉංග්‍රීසියෙන් ලියන්න. සිංග්ලිෂ් වලින් ලිවීමෙන් වලකින්න. 
+                <div v-if="details.length < 10" class="file-details"ඉහත පොත ඔබ කියවා ඇත්නම් හෝ වැදගත් බාහිර තොරතුරු දන්නවා නම් 
+                    මෙම පිටුවට ගොඩවදින අයට දැනගැනීම පිණිස මෙතැන පළ කිරීමට උචිත විස්තරයක් ලියා අප වෙත වි-තැපෑලක් එවන්න. 

```

Comment:
ඉහත, ඇත්නම්, මෙම, පළ කිරීමට these are ok. 
I would suggest to keep the rest unchanged.